DeAceroDLabs/marketplace-frontend
# Fix/dmp 739 Aligned cards on search and notification

## What is done

[[DMP-739] Aligned cards on search and notification](https://deacero.atlassian.net/browse/DMP-739)

## How is done

- Added useMediaQuery hook to set a number of cards per row on search.
- Updated notification scss.

## How to test

1. Run the project via `npm start`. Set your window width to 1250px.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. You should see the notification aligned.
5. Search a product, you should see the 3 cards per row.

## Evidence

### Before
![image](https://user-images.githubusercontent.com/81311646/208727884-3e8eeafc-1f0b-478b-ac7d-39a46c2d24fd.png)

![image](https://user-images.githubusercontent.com/81311646/208728278-83de6b10-09ca-4a67-a1c0-7831bcf16425.png)

### After
![image](https://user-images.githubusercontent.com/81311646/208729498-c211f08f-a5f9-46ff-8caa-44da616e4aae.png)
![image](https://user-images.githubusercontent.com/81311646/208729572-9543d55a-7f2f-4d6e-a811-e22eba26760b.png)
